ApplicationLibCode/ProjectDataModel/Annotations/RimTimeAxisAnnotation.cpp

```diff
@@ -139,3 +149,19 @@ void RimTimeAxisAnnotation::initAfterRead()
         setDefaultColor();
     }
 }
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+QString RimTimeAxisAnnotation::timeFormatString( const QString& customFormatString ) const
```

Comment:
This does produce a dateTimeFormatString, not a timeFormatString? Maybe change the name since we separate the two type later in the method?